go/extractor/project/project.go

```diff
@@ -38,6 +38,9 @@ type GoModule struct {
 
 // Tries to find the Go toolchain version required for this module.
 func (module *GoModule) RequiredGoVersion() util.SemVer {
+	if module.Module != nil && module.Module.Toolchain != nil {
+		return util.NewSemVer(module.Module.Toolchain.Name)
+	}
 	if module.Module != nil && module.Module.Go != nil {
```

Comment:
Massive nitpick: it would be consistent with the other `RequiredGoVersion` in the first commit if you make this an `else if`. It will also be slightly more compact.

Reply:
Makes sense. I think it probably used `else if` originally, but then turned into this while resolving merge conflicts.